src/core/thread.d

```diff
@@ -5628,3 +5631,187 @@ version (Windows)
 else
 version (Posix)
     alias ThreadID = pthread_t;
+
+///////////////////////////////////////////////////////////////////////////////
+// lowlovel threading support
+private
+{
+    __gshared size_t ll_nThreads;
+    __gshared ThreadID* ll_pThreads;
+
+    __gshared align(Mutex.alignof) void[__traits(classInstanceSize, Mutex)] ll_lock;
```

Comment:
Note: this is alignment is insufficient for targets with strict alignment requirements.

Reply:
Yes, will need to be updated in lock-step with Thread._locks.